src/cascadia/TerminalControl/IControlAppearance.idl

```diff
@@ -11,6 +11,8 @@ namespace Microsoft.Terminal.Control
         Windows.UI.Xaml.Media.Stretch BackgroundImageStretchMode;
         Windows.UI.Xaml.HorizontalAlignment BackgroundImageHorizontalAlignment;
         Windows.UI.Xaml.VerticalAlignment BackgroundImageVerticalAlignment;
+        Boolean IntenseIsBold;
+        // IntenseIsBright is in Core Appearance

```

Comment:
technically both should be there, since eventually we'll push down the renderer right?

Reply:
IMO eventually is not _today_, and I'd rather be self-consistent with the state of the world and not what we expect the world to be (which may lead to confusion)